media-plugins/mythplugins/mythplugins-30.0.ebuild

```diff
@@ -0,0 +1,167 @@
++mythmusic +mythnetvision +mythnews +mythweather +mythzmserver +mythzoneminder"
+IUSE="${MYTHPLUGINS} alsa cdda cdr exif fftw +hls ieee1394 libass +opengl raw +theora +vorbis +xml xvid"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/openssl:=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtopengl:5
+	dev-qt/qtsql:5
+	media-libs/freetype:2
+	media-libs/libpng:0=
+	sys-apps/util-linux
+	virtual/libudev:=
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXinerama
+	x11-libs/libXrandr
+	x11-libs/libXv
+	x11-libs/libXxf86vm
+	alsa? ( >=media-libs/alsa-lib-1.0.24 )
+	hls? (
+		media-libs/faac
+		media-libs/libvpx:=
+		>=media-libs/x264-0.0.20111220:=
+		>=media-sound/lame-3.93.1
+	)
+	ieee1394? (
+		>=media-libs/libiec61883-1.0.0
+		>=sys-libs/libavc1394-0.5.3
+		>=sys-libs/libraw1394-1.2.0
+	)
+	libass? ( >=media-libs/libass-0.9.11:= )
+	mytharchive? (
+		app-cdr/dvd+rw-tools
+		dev-python/pillow
+		media-video/dvdauthor
+		media-video/mjpegtools[png]
+		media-video/transcode
+		virtual/cdrtools
+	)
+	mythbrowser? ( dev-qt/qtwebkit:5 )
+	mythgallery? (
+		media-libs/tiff:0
+		opengl? ( virtual/opengl:= )
+		exif? ( >media-libs/libexif-0.6.9:= )
+		raw? ( media-gfx/dcraw )
+	)
+	mythgame? ( !syslibs/zlib[minizip] )
+	mythmusic? (
+		>=media-libs/flac-1.1.2
+		media-libs/libogg
+		>=media-libs/libvorbis-1.0
+		>=media-libs/taglib-1.6
+		>=media-sound/lame-3.93.1
+		fftw? ( sci-libs/fftw:3.0= )
+		opengl? ( virtual/opengl )
+		cdda? (
+			dev-libs/libcdio:=
+			cdr? ( virtual/cdrtools )
+		)
+	)
+	mythnetvision? (
+		${PYTHON_DEPS}
+		dev-python/pycurl
+		dev-python/lxml
+		dev-python/mysql-python
+		dev-python/oauth
+	)
+	mythweather? (
+		dev-perl/Date-Manip
+		dev-perl/XML-Simple
+		dev-perl/XML-XPath
+		dev-perl/DateTime
+		dev-perl/Image-Size
+		dev-perl/DateTime-Format-ISO8601
+		dev-perl/SOAP-Lite
+		dev-perl/JSON
+	)
+	theora? ( media-libs/libtheora )
+	xml? ( >=dev-libs/libxml2-2.6.0:= )
+	xvid? ( >=media-libs/xvid-1.1.0 )
+"
+RDEPEND="${DEPEND}
+	!<media-tv/mythtv-${PV}[alsa=,hls=,ieee1394=,libass=,python,theora=,vorbis=,xml=,xvid=]
+	!media-plugins/mytharchive
+	!media-plugins/mythbrowser
+	!media-plugins/mythgallery
+	!media-plugins/mythgame
+	!media-plugins/mythmovies
+	!media-plugins/mythmusic
+	!media-plugins/mythnetvision
+	!media-plugins/mythnews
+	!media-plugins/mythweather
+	!media-plugins/mythzmserver
+	!media-plugins/mythzoneminder
+"
+REQUIRED_USE="
+	mythmusic? ( vorbis )
+	mythnetvision? ( ${PYTHON_REQUIRED_USE} )
+	mythnews? ( mythbrowser )
+"
+
+# mythtv and mythplugins are separate builds in the github MythTV repository
+S="${WORKDIR}/${P}/mythplugins"
+
+DOC_CONTENTS="
+All plugins are installed by default. Disable unneeded plugins individually with USE flags:
+-mytharchive -mythbrowser -mythgallery -mythgame -mythmusic \
+-mythnetvision -mythnews -mythweather -mythzmserver -mythzoneminder
+"
+
+src_prepare() {
+	default
+	sed -i '1i#define OF(x) x' mythgame/mythgame/external/ioapi.h
```

Comment:
Needs to be appended with `|| die`

Reply:
Good idea.

Reply:
if the file may not exist, then check if the file exists before the `sed`. external commands must be appended with `|| die`

Reply:
Remove it altogether.  There is no `mythgame/mythgame/external/` directory in the source tree.